src/app/zkapp_test_transaction/zkapp_test_transaction.ml

```diff
@@ -53,16 +53,6 @@ module Flags = struct
 end
 
 let create_zkapp_account =
-  let create_command ~debug ~sender ~sender_nonce ~fee ~fee_payer
```

Comment:
Why inline?

Reply:
Sorry, this is the result of applying a vim macro in places where it shouldn't be applied. Some of these were intentionally inlined due to scoping requirements for the config, the ones you have hlighted are a mistake

Reply:
should be fixed now